protoc-gen-openapiv2/internal/genopenapi/types.go

```diff
@@ -54,6 +61,7 @@ type extension struct {
 type openapiSwaggerObject struct {
 	Swagger             string                              `json:"swagger"`
 	Info                openapiInfoObject                   `json:"info"`
+	Tags                []openapiTagObject                  `json:"tags"`
```

Comment:
Making this `omitempty` should omit empty objects.

Reply:
Done. A few other `omitempty` tags also added where applicable.